tutorials/eth-api/chat-gpt.md

```diff
@@ -0,0 +1,166 @@
+
+```
+I would like to create an ERC-20 token called "KevinToken" with the symbol "KEV" and an initial supply of 40000000.
+```
+
+![ChatGPT's 1st response](/images/tutorials/eth-api/chatgpt/chatgpt-4.png)
+
+This is a great start. ChatGPT has produced for us a simple, yet functional ERC-20 token meeting all of the parameters that we have specified. It also clarified how it created the ERC-20 token contract using the [OpenZeppelin standard](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol){target=_blank} and where the initial token supply is directed to. Finally, it reminds us that this is just a start and there may be other considerations we wish to implement, like minting and burning. 
+
+ChatGPT is perfectly happy to revise and expand upon the contract that was created. As long as you stay within the same chat window (i.e. don't click new chat), ChatGPT will be aware of its prior output. As an example, let's now ask ChatGPT to revise our token to be both mintable and burnable: 
+
+```
+This looks great, but I'd really like my ERC-20 to be both mintable and burnable. 
+```
+
+ChatGPT is happy to oblige. Notice how it maintains the parameters we specified originally, namely the token name and symbol. 
+
+![ChatGPT's 2nd response](/images/tutorials/eth-api/chatgpt/chatgpt-5.png)
+
+## Preparing Deployment Instructions {: #preparing-deployment-instructions }
+
+This section is named carefully to avoid implying that ChatGPT will be doing the deploying for us. ChatGPT does not have internet access and cannot interact with blockchain networks directly, but it can give us detailed instructions explaining how we can do so ourselves. Let's ask ChatGPT for instructions on deploying the recently created ERC20 contract. For this example, let's ask ChatGPT for [Hardhat deployment instructions](/builders/build/eth-api/dev-env/hardhat/){target=_blank}:
+
+```
+I would like to use Hardhat to compile and deploy this smart contract to the Moonbase Alpha network.  
+```
+
+![ChatGPT's 3rd response](/images/tutorials/eth-api/chatgpt/chatgpt-6.png)
+
+And to no surprise, ChatGPT provides us with a detailed series of deployment steps from installation instructions to a full deployment script. Note that it even remembers a detail in our first prompt that wasn't important until now. In our initial prompt, we asked for our token to have an initial supply of `400000000`, and ChatGPT included this parameter in the deployment script it generated. 
+
+Another observation is that the RPC URL it generated is outdated, although still functional. The current RPC URL for Moonbase Alpha is `{{ networks.moonbase.rpc_url }}`. This oversight is due to ChatGPT's knowledge cutoff date of September 2021, before the updated RPC URL was published.
```

Comment:
I would swap the last two sentences around and wrap the Moonbase Alpha RPC in 3 back ticks so it can be easily copied